src/bci_build/package/__init__.py

```diff
@@ -1325,6 +1329,8 @@ def title(self) -> str:
 
 @dataclass
 class OsContainer(BaseContainerImage):
+    logo_url: str = ""
```

Comment:
Wouldn't it make sense for these to use the BCI logo too?

Reply:
The challenge I had is that OsContainer is used for a mix of containers that are not BCI, like for example the LTSS or fips containers (or opensuse containers). actually the opensuse issue is general, so I need to find a different solution :/